Formula/n/netatalk.rb

```diff
@@ -105,8 +109,21 @@ def caveats
   end
 
   test do
-    system sbin/"netatalk", "-V"
+    pidfile = var/"run/netatalk#{".pid" if OS.mac?}"
+    port = free_port
+    (testpath/"afpd.conf").write <<~EOS
+      [Global]
+      afp port = #{port}
+    EOS
     system sbin/"afpd", "-V"
-    assert_empty shell_output(sbin/"netatalk")
+    system sbin/"netatalk", "-V"
+    fork do
+      system sbin/"netatalk", "-d", "-F", "afpd.conf"
+    end
+    sleep 1
```

Comment:
```suggestion
    sleep 1 until pidfile.exist?
```
Probably a good proxy for the service to be ready.